ml-proto/src/script.ml

```diff
@@ -71,6 +75,42 @@ let run_command cmd =
       Error.error cmd.at "assertion failed"
     end
 
+  | AssertHeapEq (offset, expected) ->
+    let m = match !current_module with
+      | Some m -> m
+      | None -> Error.error cmd.at "no module defined to examine heap"
+    in
+    let mem = Eval.get_module_memory m in
+    let count = String.length expected in
+
```

Comment:
Style nit: Ocaml code does not typically use empty lines in places like this (similarly below)